### PR TITLE
:email: replace e-mail with email

### PR DESCRIPTION
**Pull request by mairaw**
I've noticed that PR #4174 changed e-mail to email and when I checked our style guide, it says that email shouldn't be hyphened. So changing this across our repo.

/cc @scottaddie @Rick-Anderson in case you wanna do that for asp.net docs too

### CODE REVIEW

File: docs/framework/wpf/app-development/toc.md [hunk 1]
```diff
@@ -29,7 +29,7 @@
 #### [Get the Return Value of a Page Function](how-to-get-the-return-value-of-a-page-function.md)
 #### [Navigate Forward or Back Through Navigation History](how-to-navigate-forward-or-back-through-navigation-history.md)
 #### [Determine If a Page is Browser Hosted](how-to-determine-if-a-page-is-browser-hosted.md)
-#### [Use mailto: to Send Mail From a Page](how-to-use-mailto-to-send-mail-from-a-page.md)
+#### [Use mailto: to Send Mail From a Page](how-to-usemailto-to-send-mail-from-a-page.md)
```
Comment (rpetrusha) [inline docs/framework/wpf/app-development/toc.md]: This was an inadvertent change that's resulted in a broken link.

**Review by rpetrusha — APPROVED** [review]
This looks good, @mairaw, except for one broken link introduced as a result of the change. In docs/framework/wpf/app-development/toc.md:
"how-to-use-mailto-to-send-mail-from-a-page.md" was inadvertently changed ("...to-usemailto-to...")

**Comment by mairaw** [comment]
Good catch @rpetrusha! I've fixed it. Thanks.

**Comment by Rick-Anderson** [comment]
The build report should reflect broken links - right? Hopefully @guardrex  PR on the same didn't break any links.

**Comment by mairaw** [comment]
Yes @Rick-Anderson. It did catch.

**Comment by guardrex** [comment]
I was fairly careful not to do that. I scanned them twice. I've made a note to review my PR to confirm today.

**Pull request merged**